tests/agents/json-specs/span_types.json

```diff
@@ -20,32 +20,37 @@
           "java"
         ]
       },
+      "internal": {
+        "__description": "Application generic internal span for controller/handler/processing delegation",
+        "__used_by": [
+        ]
+      },
       "controller": {
-        "__description": "Application controller actions",
+        "__description": "Deprecated: use app/internal instead",
```

Comment:
@SylvainJuge Is there a reason you wrote `app/internal` and not 'app.internal`? I feel like the latter makes it a bit clearer what you're referring to.

Reply:
It's just the habit to refer to `type/subtype`, we can definitely change it to `app.internal` or be even more explicit with `type = 'app' and subtype = 'internal'`

Reply:
I like the `.` notation too. I believe we're using that in the Go agent to split type/subtype.